### PR TITLE
added distance to price conversion

### PR DESCRIPTION
**Pull request by sudheep2**

### CODE REVIEW

File: src/main/java/com/nsdl/beckn/init/service/InitServiceSeller.java [hunk 1]
```diff
@@ -178,13 +178,43 @@ private Price calculatePrice(Order order){
         order.getItems().forEach((item) -> {
             totalPrice.set(totalPrice.get() + item.getQuantity().getCount()*10);
         });
-        totalPrice.set(totalPrice.get()*getDistanceMultiplier(order.getFulfillment()));
+        totalPrice.set(totalPrice.get()+7*getDistanceMultiplier(order.getFulfillment()));
         price.setValue(totalPrice.get());
```
Comment (sakamotojin) [inline src/main/java/com/nsdl/beckn/init/service/InitServiceSeller.java]: why multiplied with 7

Comment (sudheep2) [reply src/main/java/com/nsdl/beckn/init/service/InitServiceSeller.java]: milage of a typical carrier

Comment (sudheep2) [reply src/main/java/com/nsdl/beckn/init/service/InitServiceSeller.java]: i mean cost per km

File: src/main/java/com/nsdl/beckn/init/service/InitServiceSeller.java [hunk 2]
```diff
@@ -178,13 +178,43 @@ private Price calculatePrice(Order order){
         order.getItems().forEach((item) -> {
             totalPrice.set(totalPrice.get() + item.getQuantity().getCount()*10);
```
Comment (sakamotojin) [inline src/main/java/com/nsdl/beckn/init/service/InitServiceSeller.java]: let's cal the cost per item here

**Pull request closed without merging**